src/wizards/tokenSwapDealWizard/stages/submitStage/daoInformation/daoInformation.html

```diff
@@ -0,0 +1,110 @@
+<template>
```

Comment:
basically copy pasted from @IonelLupu

Reply:
would it be reasonable to make this a reusable component, used in both places?  Or are there too many differences?

Reply:
At time of implementation, the mocks were not the same.
Now, I asked Jaap, and both places are in sync.

I wanted to avoid too many unrelated changes in this PR.
But would defo want to refactor in a follow up

Reply:
OK can you please create a ticket for that?  @hiaux0

Reply:
#287